web/src/server/db/db.ts

```diff
@@ -0,0 +1,74 @@
+import * as r from "rethinkdb";
+
+// Object to store the connection
+const dbConfig = {
+  host: "localhost",
+  port: 28015,

```

Comment:
You already have, an `env.js` to handle your conifgs. Please, move these inside `env.js`!